rosidl_parser/rosidl_parser/definition.py

```diff
@@ -436,6 +436,21 @@ def get_annotation_values(self, name):
         """
         return [a.value for a in self.annotations if a.name == name]
 
+    def get_comment_lines(self):
+        """
+        Get the comment lines of the annotatable.
+
+        :returns: a list of comment lines
+        """
+        comments = [
+            x['text'] for x in self.get_annotation_values('verbatim') if
+            'language' in x and 'text' in x and x['language'] == 'comment'
+        ]
+        lines = []
+        for comment in comments:
+            lines.extend(comment.split('\n'))
```

Comment:
@ivanpauno nit^N: 
```suggestion
            lines.extend(comment.splitlines())
```

Reply:
Done in https://github.com/ros2/rosidl/pull/593/commits/1b09d4d546ef3b5f7c225a4a3eb16b436741ebe4